MacLinux/README.txt

```diff
@@ -45,7 +46,7 @@ Rules
 - Tha lambda function $RULE_NAME.zip needs to be in the same directory as the createRule script
 - Create $RULE_NAME.zip from files in $RULE_NAME directory
 
-Test
+Test // DO NOT USE, STILL BETA

```

Comment:
For the Test module. I believe we need to review the approach, it is not friendly for the development of new rules. I put this comment to avoid people to try to hard.